### PR TITLE
Better styles:)

### PR DESCRIPTION
**Pull request by grigory-bogush**
trying to make application look a bit better

### CODE REVIEW

File: app/javascript/packs/src/new_bootstrap_navar.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// --- Generated ---
+$bgDefault      : #c5e1a5;
+$bgHighlight    : #94af76;
+$colDefault     : #000000;
+$colHighlight   : #424242;
+
+.navbar-custom {
+    background-color: $bgDefault;
```
Comment (DmitryLeviossa) [inline app/javascript/packs/src/new_bootstrap_navar.scss]: use 2 spaces in tab

File: app/javascript/packs/src/new_bootstrap_navar.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+/* change the brand and text color */
+.navbar-custom .navbar-brand,
```
Comment (DmitryLeviossa) [inline app/javascript/packs/src/new_bootstrap_navar.scss]: use inheritance (Block in block)

Comment (DmitryLeviossa) [reply app/javascript/packs/src/new_bootstrap_navar.scss]: https://sass-scss.ru/guide/

**Review by DmitryLeviossa — CHANGES_REQUESTED** [review]

File: app/javascript/packs/src/new_bootstrap_navar.scss (hunk 1 above)
Comment (grigory-bogush) [reply app/javascript/packs/src/new_bootstrap_navar.scss]: done

File: app/javascript/packs/src/new_bootstrap_navar.scss (hunk 2 above)
Comment (grigory-bogush) [reply app/javascript/packs/src/new_bootstrap_navar.scss]: done:)

File: app/javascript/packs/src/new_bootstrap_navar.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  /* change the link color */
+  .navbar-nav {
+    .nav-link {
+      @extend %default-color;
+    }
+  }
+  /* change the color of active or hovered links */
+  .nav-item.active {
```
Comment (DmitryLeviossa) [inline app/javascript/packs/src/new_bootstrap_navar.scss]: use pseudo classes

File: app/javascript/packs/src/style.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+.content-div {
+  background-color: $bgDefault;
+  margin: 30px 10px;
+  padding: 15px 20px;
+  border-radius: 25px;
+}
+.custom-table {
+  text-align: center;
+}
+.custom-table thead {
```
Comment (DmitryLeviossa) [inline app/javascript/packs/src/style.scss]: inheritance

**Review by DmitryLeviossa — CHANGES_REQUESTED** [review]

**Review by DmitryLeviossa — APPROVED** [review]

**Pull request merged**